<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InducedPolarizationClass</class>
 <widget class="QDialog" name="InducedPolarizationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>可视化多级界面</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="100">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_leftDown" native="true"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="100">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="ViewerWidget3d" name="widget_leftUpper" native="true"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="100">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_rightDown" native="true"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="100">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_rightUpper" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewerWidget3d</class>
   <extends>QWidget</extends>
   <header>include\3ddisplay\ViewerWidget3d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
